go.mod

```diff
@@ -5,8 +5,7 @@ require gopkg.in/yaml.v2 v2.2.1
 require (
 	github.com/google/gofuzz v1.0.0
 	github.com/json-iterator/go v1.1.6
-	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
-	github.com/modern-go/reflect2 v1.0.1 // indirect
+	sigs.k8s.io/structured-merge-diff v1.0.2
```

Comment:
I don't know if you rebased or what, but you're going to have fun fixing these :-)

Reply:
I did rebased, but not a clue what happened there.
Will clean up